templates/vue/src/utils/Helpers.js

```diff
@@ -187,4 +187,10 @@ export default class Helpers {
 
     return false
   }
+
+  /**
+   * A "no-op" - does nothing.
+   * Useful as a default value when passing functions as arguments.
+   */
+  static noop() {}
```

Comment:
Not sure if we need this since calling it takes more work than just implementing it in-place.

Reply:
I'm not sure if there would be much difference performance wise. Your other suggestion would make this part unnecessary though so I can take it off.